apps/api/src/app/controllers/command/__snapshots__/command-payload-schemas.spec.ts.snap

```diff
@@ -1340,5 +1340,105 @@ exports[`command payload schemas Command payload schema should match the snapsho
     },
     "type": "CREATE_PLAYLIST",
   },
+  {
+    "description": "creates a note about this particular resource",
+    "label": "Create Note",
+    "schema": {
+      "aggregateCompositeIdentifier": {
+        "complexDataType": "NESTED_TYPE",
+        "description": "system-wide unique identifier for the edge connection (note) being created",
+        "isArray": false,
+        "isOptional": false,
+        "label": "Composite Identifier",
+        "schema": {
+          "id": {
+            "coscradDataType": "UUID",
+            "description": "unique identifier",
+            "isArray": false,
+            "isOptional": false,
+            "label": "ID",
+          },
+          "type": {
+            "coscradDataType": "NON_EMPTY_STRING",
+            "description": "song",
+            "isArray": false,
+            "isOptional": false,
+            "label": "type",
+          },
+        },
+      },
+      "resourceCompositeIdentifier": {
+        "complexDataType": "NESTED_TYPE",
+        "description": "system-wide unique identifier for the resource about which we are making a note",
+        "isArray": false,
+        "isOptional": false,
+        "label": "CompositeIdentifier",
+        "schema": {
+          "id": {
+            "coscradDataType": "UUID",
+            "description": "unique identifier",
+            "isArray": false,
+            "isOptional": false,
+            "label": "ID",
+          },
+          "type": {
+            "coscradDataType": "NON_EMPTY_STRING",
+            "description": "song",
+            "isArray": false,
+            "isOptional": false,
+            "label": "type",
+          },
+        },
+      },
+      "resourceContext": {
+        "complexDataType": "UNION_TYPE",
+        "description": "the context for the resource you are making a note about",
```

Comment:
Detail, but the grammar doesn't match up ("about which")

Reply:
Looks like it was removed anyway, but good catch!